README.md

```diff
@@ -30,21 +59,103 @@ jobs:
   transition-jira-ticket:
     runs-on: ubuntu-20.04
     steps:
-      - name: 'Transition Jira Ticket to Deployed Status'
+      - name: Transition Jira Ticket to Deployed Status
         # You may also reference just the major or major.minor version
-        uses: im-open/transition-jira-tasks-by-query@v1.1.2
+        uses: im-open/transition-jira-tasks-by-query@v2.0.0
+        id: transition
         with:
-          jira-username: 'some-user'
-          jira-password: ${{ secrets.JIRA_USER_PASSWORD }}
-          domain-name: 'jira.com'
-          transition-name: 'Deployed'
+          jira-username: ${{ vars.JIRA_USERNAME }}
+          jira-password: ${{ secrets.JIRA_PASSWORD }}
+          domain-name: jira.com
+          transition-name: Deployed
           
           jql-query: 'issuekey=PROJ-12345'
           # jql-query: "filter='My Filter Name' AND issuekey=PROJ-12345"
           # jql-query: 'component IN ("System Infrastructure') AND 'Deployment Version' ~ 'v1.2.1''
+          # issues: TW-1234, TW-4455
+          # issues: |
+          #   TW-1234
+          #   TW-4455
           # etc.
+
+          # If you want to update or overwrite fields, you can use the following inputs:
+          # update-fields: |
+          #   {
+          #     "customfield_12345": "some value"
+          #   } 
+
+      # Some issue types don't have a transition status like others.  
+      # In those cases, where you want to still transition, a second step will need to be invoked.  
+      # You can pass in the `unavailable-issues` output to transition those remaining issues.
+      - name: Transition Remaining Jira Tickets to Closed Status
+        uses: im-open/transition-jira-tasks-by-query@v2.0.0
+        with:
+          jira-username: ${{ vars.JIRA_USERNAME }}
+          jira-password: ${{ secrets.JIRA_PASSWORD }}
+          domain-name: jira.com
+          transition-name: Closed
+          issues: ${{ steps.transition.outputs.unavailable-issues }}
+```
+
+## Updating Fields
+
+They are two different ways to update fields.  Passing a `update-fields` input or an `process-operations` input. You may not specify the same field name in both the `update-fields` and `process-operations` inputs.
+
+If you are unsure what field names to use, run the [Get-Jira-Issue](../main/Test-Get-Jira-Issue.ps1) script locally from your machine using your ExtendHealth username and password.
```

Comment:
Again, let's avoid using our terminology. Everyone should be familiar with logging into Jira and which credentials are required anyway.